uint64   time
uint64   devnum
float32[] setpoints
float32[] measured
float32[] input_pressure